components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -868,13 +867,12 @@ def start_webui(
     # Read, update, and write back client's and server's settings.json
     clp_db_connection_params = clp_config.database.get_clp_connection_params_and_type(True)
     table_prefix = clp_db_connection_params["table_prefix"]
-    if StorageEngine.CLP_S == clp_config.package.storage_engine:
-        table_prefix = f"{table_prefix}{CLP_DEFAULT_DATASET_NAME}_"
+    dataset = CLP_DEFAULT_DATASET_NAME if StorageEngine.CLP_S == clp_config.package.storage_engine else None
```

Pylint (3.3.7)

[convention] Line too long (108/100)

(C0301)